gulp/tasks/test.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var $ = require( 'gulp-load-plugins' )();
+var protractor = require( 'gulp-angular-protractor' );
+var exec = require( 'child_process' ).exec;
+var config = require( '../config' ).test;
```

Comment:
Should be `var test`, see https://nodejs.org/api/child_process.html for example of naming style.